assets/js/blocks/products-by-attribute/block.js

```diff
@@ -16,7 +16,7 @@ import {
 	withSpokenMessages,
 } from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
-import Gridicon from 'gridicons';
+import { Icon, tags } from '@woocommerce/icons';
```

Comment:
In the design posted in #1418, _Products by Attribute_ icon looks like this:
![imatge](https://user-images.githubusercontent.com/3616980/73080873-d36ffd00-3ec6-11ea-8c23-3fd66f0592fb.png)

But in this branch it is:
![imatge](https://user-images.githubusercontent.com/3616980/73080829-c18e5a00-3ec6-11ea-9a0a-cd6fdc85b071.png)

Is that change in purpose?

Reply:
yes, this confused me first, while the design that Josh provided used the same icon twice (in filter by attributes and product by attributes, the files provided are not the same, the current design is according to the files)